pkg/config/admin-server/config.go

```diff
@@ -0,0 +1,139 @@
+package admin_server
+
+import (
+	"github.com/Kong/kuma/pkg/config"
+	"github.com/pkg/errors"
+)
+
+func DefaultAdminServerConfig() *AdminServerConfig {
+	return &AdminServerConfig{
+		DataplaneTokenWs: DefaultDataplaneTokenWsConfig(),
+		Local:            DefaultLocalAdminServerConfig(),
+		Public:           DefaultPublicAdminServerConfig(),
+	}
+}
+
+var _ config.Config = &AdminServerConfig{}
+
+// Admin Server configuration
+type AdminServerConfig struct {
+	// Configuration for Dataplane Token Webservice
+	DataplaneTokenWs *DataplaneTokenWsConfig `yaml:"dataplaneTokenWs"`
```

Comment:
Name `dataplaneTokenWs` that will appear in YAML and env variables is too implementation-specific.

Let's use the same approach as in `Catalog`.

E.g.,

```
type AdminServerConfig struct {
   Apis *ApisConfg
}

type ApisConfig struct {
  DataplaneToken *DataplaneTokenApiConfig
}

type DataplaneTokenApiConfig struct {}
```